### PR TITLE
Footer & impressum & more

### LINKED ISSUE #93: Impressum page
Basics are done with the pictures, come up with some info about the project and KirDev

### PR DESCRIPTION
**Pull request by Tschonti**
image

image

Video about the hover effects in the #arculat slack channel

Also improved the text on the impressum page, but it's not final

Closes #93 and #94

### CODE REVIEW

File: src/components/commons/Footer.tsx [hunk 1]
```diff
@@ -1,20 +1,93 @@
...
-        <Text mt={2} textAlign={{ base: 'center', sm: 'right' }}>
-          &copy; {new Date().getFullYear()} • Kir-Dev
-        </Text>
-      </Flex>
-    </Container>
-  </Box>
-)
+export const Footer: FC = () => {
+  const kirDevLogo = useColorModeValue('/img/kirdev.svg', '/img/kirdev-white.svg')
+  const hkLogo = useColorModeValue('/img/hk.svg', '/img/hk-white.svg')
```
Comment (berenteb) [inline src/components/commons/Footer.tsx:10]: WHy aren't you manipulating the SVG itself with inline colors or css?

**Review by berenteb — CHANGES_REQUESTED** [review]

File: src/components/commons/Footer.tsx (hunk 1 above)
Comment (Tschonti) [reply src/components/commons/Footer.tsx:10]: this was easier 😄

File: src/components/commons/Footer.tsx [hunk 2]
```diff
@@ -1,20 +1,93 @@
...
+                <FaGithub size={25} />
+              </Link>
+              <Link href="https://kir-dev.hu" isExternal _hover={{ color: customTheme.colors.kirDev }}>
+                <FaGlobe size={25} />
+              </Link>
+              <Link href="https://www.facebook.com/kirdevteam" isExternal _hover={{ color: customTheme.colors.kirDev }}>
+                <FaFacebook size={25} />
+              </Link>
+              <Link href="https://www.instagram.com/kir.dev/" isExternal _hover={{ color: customTheme.colors.kirDev }}>
+                <FaInstagram size={25} />
```
Comment (DannySS123) [inline src/components/commons/Footer.tsx:85]: perhaps a socialLinkButton component? :D

File: src/pages/impressum/ImpressumPage.tsx [hunk 3]
```diff
@@ -1,23 +1,46 @@
...
       </Heading>
-      <ReactMarkdown>
-        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text
-        ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived
-        not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the
-        1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like
-        Aldus PageMaker including versions of Lorem Ipsum.
-      </ReactMarkdown>
+      <Text textAlign="justify">
+        Az alkalmazást a HK felkérésére Kir-Dev webfejlesztő kör készítette. Mint ahogy az összes többi projektünk, ez is nyílt forráskódú.
```
Comment (DannySS123) [inline src/pages/impressum/ImpressumPage.tsx:15]: Sounds a bit strange, maybe:
Az alkalmazást a HK felkérésére a Kir-Dev webfejlesztő kör készítette. (Kir-Dev előtt "a")

**Review by DannySS123 — CHANGES_REQUESTED** [review]
Lookin 🔥🔥🔥🔥🔥
Would have liked the color change in th logos in light mode as well, but I guess it can be counted as an easter egg in dark mode :D

And yes, for the impressum to be completed I think we'll nedd something under the images of the developers as well. But it's good for now.

**Comment by Tschonti** [comment]
More styling changes:

- changed the avatar of consultations, now it's related to the major of the subject
image

- fixed scrollbar. It was previously invisble on the whole page, because it was using non-existent chakra variables (shouldn't have copied mindlessly from Trisz' repo xd). Couldn't find a way to change the color in dark mode, so the brand color doesn't look good, I think gray works fine.
image

File: src/components/commons/Footer.tsx (hunk 1 above)
Comment (Tschonti) [reply src/components/commons/Footer.tsx:11]: I'm not really in the mood for editing the SVGs, so if it's okay with you, I wouldn't change this

Comment (berenteb) [reply src/components/commons/Footer.tsx:11]: Okay OFC

**Review by berenteb — APPROVED** [review]

**Review by DannySS123 — APPROVED** [review]
✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️

**Pull request merged**